src/xtgeo/grid3d/grid.py

```diff
@@ -2287,7 +2435,7 @@ def reverse_row_axis(self, ijk_handedness=None):
         _grid_etc1.reverse_row_axis(self, ijk_handedness=ijk_handedness)
         self._tmp = {}
```

Comment:
I'm sure you noticed this yourself but beginning to feel there must be a better way to handle things with (or preferably without) `self._tmp`. Not relevant to this PR of course but just a general comment

Reply:
Yes the `self._tmp` is applied in `_grid_etc1.py` and `_grid3d_fence.py` to store some data to speed up execution, but I am sure this can handled in a better way.